docs/server-admin.md

```diff
@@ -0,0 +1,37 @@
+# The administration panel
+
+Iodide has an administration view under `/admin`, based on the [Django
+admin](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/) interface.
+
+To log into this interface, a user must either have the `is_staff` or `is_superuser` permission set. Most operations (including the one currently
+described here) require `is_superuser`.
+
+An existing super user can grant old users super-user permissions via
+the panel. Or, if no super user is currently configured, you can perform this operation manually via a SQL session (`./manage.py dbshell` from the console):
```

Comment:
Not a big deal, but a *make dbshell* command would be handy. :smiley:

Reply:
Feel free to send in a PR :)